### ISSUE TITLE
T_PAAMAYIM_NEKUDOTAYIM

### ISSUE THREAD

**Issue by schel4ok** [laravel/framework#15822]
- Laravel Version: 5.3.16
- PHP Version: 5.6.12
- Database Driver & Version: mysql

### Description:
I have console error message when try to publish Scout configuration
```
  [Symfony\Component\Debug\Exception\FatalErrorException]
  syntax error, unexpected '::' (T_PAAMAYIM_NEKUDOTAYIM)
```

### Steps To Reproduce:
Fresh installation of Laravel.
```
> composer create-project --prefer-dist laravel/laravel blog3
Installing laravel/laravel (v5.3.16)
...
...
> cd blog3
> composer require laravel/scout
Using version ^1.1 for laravel/scout
...
...
Generating optimized class loader
```

then I add `Laravel\Scout\ScoutServiceProvider::class,` in config/app.php configuration file:
then again go to console
```
> php artisan vendor:publish

  [Symfony\Component\Debug\Exception\FatalErrorException]
  syntax error, unexpected '::' (T_PAAMAYIM_NEKUDOTAYIM)
```

And if I comment line `Laravel\Scout\ScoutServiceProvider::class,`   php artisan vendor:publish  works fine -> no error

**Comment by jhoff** [comment]
Does adding `-vvv` to `vendor:publish` provide any more context?

**Comment by schel4ok** [comment]
no

**Comment by javiereguiluz** [comment]
@schel4ok here you have more information about this error and possible solutions for it: https://github.com/laravel/scout/issues/91

**Comment by schel4ok** [comment]
fixed in laravel/scout (v1.1.7)
